rm-services-rh-stub/.idea/misc.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.6 (census-services-stub-CBrjKAeL) (3)" project-jdk-type="Python SDK" />
```

Comment:
Yesterday I have seen Ella working on Python 3.7.1. Do we need to have/use the same version of python/java/other language on the teams members machines ?